CHANGELOG.md

```diff
@@ -1,3 +1,114 @@
+# v1.13.0 (2022-03-??)
+
+## OS Changes
+
+* Set NVMe IO request timeouts for EBS according to AWS recommendations ([#2820])
+* Add `ethtool` to release ([#2829])
+* Add `NoOpMigration` for migrations and migration-helpers
+  for when no data migration is necessary _(Also cherry-picked to v1.12.0)_ ([#2749])
+* Add info logging to migrator in order to document onging migrations ([#2751])
+* Rename `ncurses` to `libncurses` ([#2769])
```

Comment:
Is this something a consumer of Bottlerocket will care about, or just an internal detail? I think it's just internal.